.github/workflows/on-push-to-main.yml

```diff
@@ -63,4 +63,5 @@ jobs:
           RESOURCE: ./nais/nais.yaml
           VARS: ./nais/prod-sbs/default.json
           VAR: version=${{ env.IMAGE }}
-          PRINT_PAYLOAD: true
\ No newline at end of file
+          PRINT_PAYLOAD: true
+          DRY_RUN: true
```

Comment:
Hvorfor er denne satt til true? Tror det kan bli overskrevet av workflow authority.

Reply:
Planen var å ikke dytte den ut i prod uten å ha testet i q1 først. I våre apper fikser vi det ved å ha DRY_RUN til true.
Om dere har andre metoder for å oppnå det samme: vi anbefaler at den metoden brukes her :-)

Reply:
Ok, skjønner. Siden det er deploy-et til q1 og kan testes der, så kan endringen i on-push-to-main godt fjernes @theagilepadawan.